### PR TITLE
Attempt to fix insecure request warnings in Jenkins.

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1507)

### CODE REVIEW

File: tests/framework/test_reporting/aggregate_test_results.py [hunk 1]
```diff
@@ -136,7 +136,9 @@ def get_best_runs(job_name, build_nums):
     required_tests = set(sys.argv[5].split())
 
     # Fetch the downstream build info from jenkins
-    requests.packages.urllib3.disable_warnings()
+    import urllib3
```
Comment (johnsonw) [inline tests/framework/test_reporting/aggregate_test_results.py]: Does this need to be imported a second time?

File: tests/utils/http_requests.py [hunk 2]
```diff
@@ -11,12 +11,9 @@
...
 class HttpRequests(object):
-    try:
-        requests.packages.urllib3.disable_warnings()
-    except AttributeError:
-        # Running on OSX the disable_warnings is not available, so catch and deal with that. The effect of
-        # ignoring the error is additional warnings so no real risk.
-        pass
+    import urllib3
+
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```
Comment (johnsonw) [inline tests/utils/http_requests.py]: Is this something that we could do at the application level?

File: tests/framework/test_reporting/aggregate_test_results.py (hunk 1 above)
Comment (jgrund) [reply tests/framework/test_reporting/aggregate_test_results.py]: shrug... I just changed all the previous usages to this one.

File: tests/utils/http_requests.py (hunk 2 above)
Comment (jgrund) [reply tests/utils/http_requests.py]: We could probably do it with environment vars, but given this only really runs on a linux host and has hardcoded RPM deps that need to be fixed this way, I'd rather do it in code.

**Comment by jgrund** [comment]
Link showing it working: https://build.whamcloud.com/job/integration-tests-shared-storage-configuration/3799/arch=x86_64,distro=el7,label=IML/console

**Pull request merged**